program/c/src/oracle/oracle.h

```diff
@@ -21,6 +21,8 @@ extern "C" {
 #define PC_PUBKEY_SIZE       32
 #define PC_PUBKEY_SIZE_64   (PC_PUBKEY_SIZE/sizeof(uint64_t))
 #define PC_MAP_TABLE_SIZE   640
+#define PC_MAX_PUBLISHERS    256
+#define PC_MAX_SYMBOLS       1024
```

Comment:
what we do if we exceed this numbers?

Reply:
That sounds like a Pythnet v2 problem anyway

Reply:
The program doesn't allow exceeding these numbers, we will probably face a lot of issues if we decide to increase these numbers. We are hoping to go to pythnet V2 before we need to increase these numbers. From the trasactions limit that we currently have, i don't think this will an issue.
But if you think there is a chance that we exceed these number before pythnet v2, we better address it now.